File: lab1-3/lab2.java
import java.util.Scanner;
public class lab2 {
	static int gcd(int a, int b) {
		while(b != 0) {
			int t = a%b;
			a = b;
			b = t;
		}
		return a;
	}
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int a = scan.nextInt();
		int b = scan.nextInt();

		System.out.println(gcd(a, b));
	}
}

File: lab1-3/lab2.cpp
#include <stdio.h>
int gcd(int a, int b)
{
	while(b)
	{
		int t = a%b;
		a = b;
		b = t;
	}
	return a;
}
int main()
{
	int a, b;
	scanf("%d%d", &a, &b);
	printf("%d\n", gcd(a,b));
}
